aot-methods.txt

```diff
@@ -307,4 +307,34 @@ com/nokia/mid/ui/gestures/GestureEventImpl.getStartY.()I
 com/nokia/mid/ui/gestures/GestureEventImpl.getFlickDirection.()F
 com/nokia/mid/ui/gestures/GestureEventImpl.getFlickSpeed.()I
 com/nokia/mid/ui/gestures/GestureEventImpl.getFlickSpeedX.()I
-com/nokia/mid/ui/gestures/GestureEventImpl.getFlickSpeedY.()I
\ No newline at end of file
+com/nokia/mid/ui/gestures/GestureEventImpl.getFlickSpeedY.()I
+com/sun/cldc/i18n/j2me/UTF_8_Reader.prepareForNextChar.(I)V
+com/sun/cldc/i18n/j2me/UTF_8_Reader.getByteOfCurrentChar.(IZ)I
+java/lang/StringBuffer.append.(C)Ljava/lang/StringBuffer;
+java/lang/String.charAt.(I)C
+java/lang/String.length.()I
+java/lang/String.indexOf.(II)I
+java/lang/String.substring.(II)Ljava/lang/String;
+java/lang/String.<init>.(II[C)V
+java/lang/String.trim.()Ljava/lang/String;
+java/lang/String.getChars.(II[CI)V
+java/lang/StringBuffer.append.(Ljava/lang/String;)Ljava/lang/StringBuffer;
+java/lang/StringBuffer.expandCapacity.(I)V
+java/lang/String.equals.(Ljava/lang/Object;)Z
+java/lang/StringBuffer.charAt.(I)C
+java/lang/StringBuffer.length.()I
+java/lang/String.substring.(I)Ljava/lang/String;
+java/lang/StringBuffer.toString.()Ljava/lang/String;
+java/lang/String.<init>.(Ljava/lang/StringBuffer;)V
+java/lang/StringBuffer.setShared.()V
+java/lang/StringBuffer.getValue.()[C
+java/lang/StringBuffer.<init>.()V
+java/lang/String.indexOf.(Ljava/lang/String;I)I
+java/lang/String.indexOf.(Ljava/lang/String;)I
+java/lang/String.indexOf.(I)I
+java/lang/String.<init>.([CII)V
+java/lang/String.toString.()Ljava/lang/String;
+com/sun/cdc/io/j2me/file/Protocol.connect.(Ljava/lang/String;Ljava/lang/String;)V
```

Comment:
> This might be unnecessary now that @myk optimized it. It's called a lot though, so it might still be worth AOT compiling it.

I actually optimized *ensureConnected*, which calls *connect*, but the effect is the same, since *ensureConnected* used to call *connect* unconditionally, and now it only calls it rarely.